Documentation/ListModule/UsingEffectively/Index.rst

```diff
@@ -5,13 +5,13 @@
 Introduction
 ============
 
-The :guilabel:`Web > List` module allows you to browse through
-each page or folder within your site and view all of the records that
-are stored within it. The :guilabel:`List` module also gives you the ability to
-create and manage records that don't have a dedicated module.
+The :guilabel:`Web > List` module allows you to browse through pages and folders
+in your site and view the records that
+are stored there. You can also use the :guilabel:`List` module to
+create and manage records without their own module.
```

Comment:
"without their own module" has another meaning than the previous "that don't have a dedicated module."

The new phrase assumes, there is an own module for a record (always), but you can use the List module instead. But this is the exception.

Reply:
@brotkrueml but even if there is a module for a record you can still use the list module?

Reply:
To me it reads: If there was not list module you would need a dedicated module for each record